src/main/python/cfn_sphere/util.py

```diff
@@ -180,6 +205,9 @@ def get_git_repository_remote_url(working_dir):
         else:
             return None
 
+def get_resources_dir():
+    script_dir = os.path.dirname(os.path.realpath(__file__))
+    return os.path.realpath(os.path.join(script_dir, "../../resources"))
 
 if __name__ == "__main__":
-    pass
\ No newline at end of file
+    print(get_resources_dir())
```

Comment:
Required?

Reply:
No, but this is to keep inline with upstream to make the next merge easier.